README.md

```diff
@@ -88,6 +88,7 @@ If you want to contribute to this list, then please read the [contributing guide
 
 ## Client libraries
 - [awaited-surrealdb](https://github.com/theopensource-company/awaited-surrealdb) - Typescript wrapper library for surrealdb.js that holds queries until connection is established.
+- [surrealdb-client](https://github.com/ri-nat/surrealdb-client) - Ruby client library with support for both HTTP and WebSocket transports.
```

Comment:
Haha we are probably going to get clashes with such names 😂 I don't think we have clashes atm, but perhaps might be worth changing the name to include the ruby indicator? For example `surrealdb-client-rb`? But happy with it as is, thank you for developing it! Will wait for @tobiemh for final approval and merge

Reply:
You make a valid point, but there is a small issue.

In Ruby, the gem name plays an important role in defining the file structure and the naming of the entry class for proper autoloading by package manager. By naming the gem `surrealdb-client`, users can expect it to be automatically loaded by Bundler as `SurrealDB::Client` without any additional hassle. For a brief overview of how naming works in Ruby, you can refer to [this StackOverflow post](https://stackoverflow.com/a/21340134).

Personally, I do not like having my main class named `SurrealDB::Client::Ruby`, nor do I want to give up on autoloading 🙃 However, it is possible to rename the repository itself, but it may end up causing more confusion than helping those searching for the gem's repository by its name.

Reply:
Yeah totally understandable - it's your project after all and the api is an important aspect. In that case happy to keep as-is, but will still wait for Tobie :) Thank you for the explanation as well! I had no idea that's how ruby worked.

Reply:
Sure, totally agree!

@tobiemh I'm OK with renaming this library to something different, if you say so. I wouldn't want to cause any headaches :)